src/main/java/bio/terra/model/DRElement.java

```diff
@@ -40,12 +40,14 @@ public boolean isLeaf() {
     // Describe generates output formatted by the object. This doesn't separate presentation from logic,
     // but it is simple. The default version displays the default information.
     public void describe(String format) {
-        if (StringUtils.equalsIgnoreCase(format, CommandUtils.CLIFormatFlags.CLI_FORMAT_TEXT.getValue())) {
-            describeText();
-        } else if (StringUtils.equalsIgnoreCase(format, CommandUtils.CLIFormatFlags.CLI_FORMAT_JSON.getValue())) {
-            describeJson();
-        } else {
-            CommandUtils.printErrorAndExit("Invalid format; only text and json are supported");
+        switch (CLIFormatFlags.lookup(format)) {
+            case CLI_FORMAT_TEXT:
+                describeText();
+                break;
+
+            case CLI_FORMAT_JSON:
+                describeJson();
+                break;
```

Comment:
I think we should have a default case in the switch that prints the error and exits.